### PR TITLE
Connect welcome view (UI) and new connection status bar

### PR DESCRIPTION
**Pull request by karolh2000**
New features:
 
- Status bar added
 - NodeConfig view removed and the functionality was moved to the status bar.
 - Bug fixes

## Description

Fixes pyrsia/pyrsia-vscode-extension#

New functionality

## Screenshots (optional)
**### FYI: The error message shown on the screen - "Not connect, ...." was updated to "Not connected,..."**

Screenshot 2023-01-30 at 1 08 37 PM

**### FYI: The error message shown on the screen - "Not connect, ...." was updated to "Not connected,..."**

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/community/get_involved/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia-vscode-extension/issues).

### Code Contributions

- [x] I've run [pre-commit.sh](https://github.com/pyrsia/pyrsia-vscode-extension/blob/main/pre-commit.sh) successfully.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,8 +42,9 @@
     "views": {
       "pyrsia": [
         {
-          "id": "pyrsia.node-config",
-          "name": "Node Status"
+          "id": "pyrsia.node-status",
+          "name": "Connect to Pyrsia",
+          "when": "pyrsia.connection.status == false || !pyrsia.connection.status"
```
Comment (fishseabowl) [inline package.json:47]: looks redundant
```suggestion
          "when": "!pyrsia.connection.status"
```

Comment (karolh2000) [reply package.json:47]: @fishseabowl Agreed, missed this one.

**Review by fishseabowl — APPROVED** [review]
lgtm

File: package.json [hunk 2]
```diff
@@ -18,6 +18,18 @@
...
   "contributes": {
+    "viewsWelcome": [
+      {
+          "view": "pyrsia.node-status",
+          "contents": "Thank you for installing the Pyrsia extension! \nIf you already have a Pyrsia node [installed and running](https://pyrsia.io/docs/tutorials/quick-installation), please connect it to VS Code:\n[Connect to Pyrsia](command:pyrsia.node-config.update-url)\n👋 Don't have a Pyrsia installed? [Download it here!](https://pyrsia.io/docs/tutorials/quick-installation)",
+          "when": "!pyrsia.connection.configured"
+      },
+      {
+        "view": "pyrsia.node-status",
+        "contents": "👋 Pyrsia node is offline? \nIf you already have a Pyrsia node [installed and running](https://pyrsia.io/docs/tutorials/quick-installation), please connect it to VS Code:\n[Connect to Pyrsia](command:pyrsia.node-config.update-url)\nDon't have a Pyrsia installed? [Download it here!](https://pyrsia.io/docs/tutorials/quick-installation)",
```
Comment (dasmanas) [inline package.json:29]:
```suggestion
        "contents": "👋 Is Pyrsia node offline? \nIf you already have a Pyrsia node [installed and running](https://pyrsia.io/docs/tutorials/quick-installation), please connect it to VS Code:\n[Connect to Pyrsia](command:pyrsia.node-config.update-url)\nDon't have a Pyrsia node installed? [Download it here!](https://pyrsia.io/docs/tutorials/quick-installation)",
```

**Review by dasmanas — APPROVED** [review]
lg with few minor comments

**Review by dasmanas — APPROVED** [review]

**Pull request merged**